parsec/core/mountpoint/manager.py

```diff
@@ -284,6 +295,8 @@ async def mountpoint_manager_factory(
     # Now is a good time to perform some cleanup in the registry
     if os.name == "nt":
         cleanup_parsec_drive_icons()
+    elif os.name == "posix" and _platform == "darwin" and os.path.isdir(base_mountpoint_path):
```

Comment:
can't just do `elif sys.platform == "darwin":` ?

and what the need for `os.path.isdir(base_mountpoint_path)` ? is there usecase where it's not the case ?

Reply:
I guessed there would possibly be situations where the mountpoint folder doesn't exist, such as the first use of Parsec on a computer.